events_task/lib/my_app_web/controllers/auth_controller.ex

```diff
@@ -13,7 +13,8 @@ defmodule MyAppWeb.AuthController do
   def create(conn, %{"user" => %{"email" => email}}) do
     case Accounts.get_or_create_by_email(email) do
       {:ok, user} ->
-        {:ok, _, _} = Guardian.send_magic_link(user)
+        {:ok, token, _} = Guardian.send_magic_link(user)
+        conn = assign(conn, :token, token)
```

Comment:
Saving data to `conn` to only be used in tests is not encouraged, especially when it's secure-sensetive data.
I see that this is used to test magic link - better approach would be to call `Guardian.send_magic_link` in test itself.